File: fast-react/src/common/enmus/document-status-enum.js
export const documentStatusEnum = {
    ALL: 100,
    EDIT: 1,
    REVIEW: 2,
    FINISH: 3,
    getName: (code) => {
        if (code === 100) return '全部'
        if (code === 1) return '编制'
        if (code === 2) return '待评审'
        if (code === 3) return '定版'
    }
}

File: fast-boot/src/main/java/com/example/fastboot/common/enums/DocumentStatusEnum.java
package com.example.fastboot.common.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

/**
 * @description:
 * @author: winl
 * @time: 2023/2/7 16:21
 */
@AllArgsConstructor
@Getter
public enum DocumentStatusEnum {
    EDIT(1,"编制"),
    REVIEW(2,"待评审"),
    FINAL(3,"定版");

    private int code;
    private String name;

}
